migration_original/ODS1Stage/tables/Base/ProviderVideo/spu_translated_ProviderVideo.sql

```diff
@@ -46,18 +58,18 @@ select_statement := $$ select
                             ifnull(json.video_LastUpdateDate, sysdate()) as LastUpdateDate,
                             mc.mediacontexttypeid
                         
-                        from raw.vw_PROVIDER_PROFILE as JSON
+                        from Cte_video as JSON
                             left join base.provider as P on p.providercode = json.providercode
                             left join base.mediavideohost as MH on mh.mediavideohostcode = json.video_REFMEDIAVIDEOHOSTCODE
                             left join base.mediareviewlevel as MR on json.video_REFMEDIAREVIEWLEVELCODE = mr.mediareviewlevelcode
                             left join base.mediacontexttype as MC on json.video_REFMEDIACONTEXTTYPECODE = mc.mediacontexttypecode
-                        where PROVIDER_PROFILE is not null
-                            and PROVIDERID is not null
+                        where 
+                            PROVIDERID is not null
```

Comment:
all of these joins can be inner joins and you can avoid have the where if null the ids